nav/nav_stack/config/global_costmap_params.yaml

```diff
@@ -8,3 +8,7 @@ global_costmap:
   # Full width and height of IGVC map
   width: 50
   height: 50
+  plugins:
+     - {name: obstacles, type: "costmap_2d::VoxelLayer", output: "screen"}
+     #- {name: virtual_layer, type: "virtual_layers::VirtualLayer", output: "screen"}
```

Comment:
Leave comment explaining why this is commented out (will be useful for future reference)